Makefile

```diff
@@ -0,0 +1,13 @@
+include env.make
+
+create-pip-env:
+	pipenv install
```

Comment:
I have to run `pipenv install --python=python3` on my setup. I'm running under WSL, and at some point pipenv install wants to execute the python.exe under Windows if I don't pass the `--python=python3` argument. Dunno why though, but passing the additional argument doesn't hurt, right?